geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -187,27 +193,49 @@ The header can be in a separate file, diff in the `*.html.ejs` file:
 
 The content will be in the `geoportal/{{package}}_geoportal/static/header.html` file.
 
+Print
+-----
 
-Information
+- With the default configuration we can easier switch from local print and mutualised print
```

Comment:
I'm not sure this should be in the changelog, as it has no direct impact on configuration

Reply:
Yes, it has an impact on the configuration, it's because of that I mention it here...